aviatrix/resource_aviatrix_aws_tgw.go

```diff
@@ -141,6 +141,13 @@ func resourceAviatrixAWSTgw() *schema.Resource {
 					},
 				},
 			},
+			"cloud_type": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      1,
+				Description:  "Type of cloud service provider, requires an integer value. Supported for AWS (1) and AWS GOV (256). Default value: 1.",
+				ValidateFunc: validation.IntInSlice([]int{1, 256}),
```

Comment:
```suggestion
				ValidateFunc: validation.IntInSlice([]int{goaviatrix.AWS, goaviatrix.AWSGOV}),
```

Could use the cloud type constants here.